GPU/GPUTracking/Base/opencl2/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ if(OPENCL2_ENABLED_SPIRV) # BUILD OpenCL2 intermediate code for SPIR-V target
   cmake_path(GET LLVM_SPIRV PARENT_PATH TMP_LLVM_SPIRV_PATH)
   add_custom_command(
       OUTPUT ${CL_BIN}.spirv
-      COMMAND PATH+=:${TMP_LLVM_SPIRV_PATH} ${LLVM_CLANG}
+      COMMAND ${CMAKE_COMMAND} -E env "PATH=${TMP_LLVM_SPIRV_PATH}:$ENV{PATH}" ${LLVM_CLANG}
```

Comment:
This will not work, since it will take the $PATH from during CMake execution, which is not necessarily the same as during `make`.
You could try with `PATH="${TMP_LLVM_SPIRV_PATH}:$PATH"`, and probably you'll have to escappe the `$` in PATH